components/geometry-merger/README.md

```diff
@@ -20,19 +20,34 @@ still be manipulated individually since this component keeps a `faceIndex` and
 
 ### API
 
-For either `geometry-merger` or `buffer-geometry-merger` components:
-
 | Property         | Description                                                                                                                                                                                                                                               | Default Value |
 | --------         | -----------                                                                                                                                                                                                                                               | ------------- |
 | preserveOriginal | Whether to remove the now-merged child goemetry or keep in scene graph. It can be useful to keep the original child entities' geometries and set their `material="visible: false"` so that we can still interact with them with colliders and raycasters. | false         |
-| materialColors | Whether to use material colors in merged geometries for the faces of the final mesh, so that colors of the faces in this final mesh resemble the colors of the original entities when they were material colors. | true |
 
 #### Members
 
 | Member      | Description                                                                                                                                                 |
 |-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| faceIndex   | Map of child A-Frame entitites Object3D UUIDs to array of face indices indiciating which faces in the merged geometries belong to the former child geometry |
-| vertexIndex   | Map of child A-Frame entitites Object3D UUIDs to array of vertex indices indiciating which vertices in the merged geometries belong to the former child geometry |
+| vertexIndex   | Map of child A-Frame entitites Object3D UUIDs to array of vertex indices indicating which vertices in the merged geometries belong to the former child geometry.<br />Each array consists of two values, indicating the start & end index for the vertices associated wit |
+
+#### Methods
```

Comment:
Modifying the colors in a BufferGeometry is quite fiddly, so I wanted to offer a simple API for these operations.